.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: Invidious CI
+
+on:
+  push:
+    branches: "master"
```

Comment:
This should apply to every branch if possible (the api-only branch was used by some people)

Reply:
Fixed. We should deprecate that branch though, it hasn't been updated since March. We can add a config option instead.

Reply:
Technically it has it's use, the problem is that I don't know how to maintain it.

Reply:
The `api-only` branch is just the `master` branch with only `/api/*` and proxy related endpoints. Once the routes are all extracted it's easy to add an `api-only` config option that disables any front-end endpoints.

If you want to manually maintain the `api-only` branch until then you can merge the current master into it and then just delete all the front-end routes from `src/invidious.cr`